src/panels/PanelBase.js

```diff
@@ -97,41 +97,45 @@ export default {
   },
   methods: {
     toggle() {
-      this.localExpanded = !this.localExpanded;
-      if (this.localExpanded) {
-        this.isCached = true;
-      }
-    },
-    expand() {
-      this.localExpanded = !this.localExpanded;
-      if (this.localExpanded) {
+      if (!this.isCached && !this.localExpanded) {
+        // Let the content load before toggling localExpanded which triggers the animation,
+        // as we need to know the scrollHeight of the content for the animation to work.
         this.isCached = true;
+        this.$nextTick(() => this.localExpanded = true);
+      } else {
+        this.localExpanded = !this.localExpanded;
       }
     },
     close() {
+      this.localExpanded = false;
       this.localMinimized = true;
     },
     open() {
-      this.localExpanded = true;
       this.isCached = true;
+      this.$nextTick(() => this.localExpanded = true);
```

Comment:
Let the dom update from loading the content trigger before triggering the animation with `localExpanded`, because we need to know the max height. ( this is called when expanding a minimized panel into full form )